### ISSUE TITLE
Patients table not reloading on Add Patient Pop-up close

### ISSUE THREAD

**Issue by manaschubby** [manaschubby/MediLink-Pro#44]
Patients table should reload after the user has added the patient. Right now we have to explicitly enter `CMD+R` or `CTRL+R` to see the new patient.

**Comment by web-dev-kushal** [comment]
I want to work on this issue..Do assign me!!

**Comment by manaschubby** [comment]
@web-dev-kushal Thank you for reaching out! We would love to have you on board.

Clone the repo and start the app on your machine. When you have fixed the issue, create a fork and open a pull request with the fix. If possible, please explain how you have fixed the issue in the Pull Request Descirption. Request a review from @manaschubby or @Shlokkzz.

I am assigning this issue to you. 

Thank for contributing to MediLink Pro!

**Comment by manaschubby** [comment]
Please reach out if you are facing any issues in understanding the code or setting up the development environment on your machine. We are here to help!

**Comment by web-dev-kushal** [comment]
I am having problem setting up.
![image](https://github.com/manaschubby/MediLink-Pro/assets/68588123/d668ff60-671d-4aad-962a-72909e5f72a0)

**Comment by manaschubby** [comment]
Looks like your React app starts successfully
Does your electron app not start?

If so, try running `npm run start` and when the browser window opens, run `npm run electron`

**Comment by web-dev-kushal** [comment]
No both has started concurrently but its not showing anything in browser and electron app.

**Comment by manaschubby** [comment]
Press `CTRL+R` on the electron app.

**Comment by Shlokkzz** [comment]
> No both has started concurrently but its not showing anything in browser and electron app.

It'll take some time. Also you can save(Ctrl+s) the main.js file to reload the app. Works for me atleast.

**Comment by manaschubby** [comment]
Hi @web-dev-kushal Please sync your fork, there are some changes regarding our migration from BrowserRouter to HashRouter. You'll be able to open the AddPatient window properly now

**Comment by web-dev-kushal** [comment]
Okay sure

**Comment by web-dev-kushal** [comment]
There is some problem connecting to mongo db

**Comment by manaschubby** [comment]
What is the issue you are facing?

**Comment by manaschubby** [comment]
There is an example.env file. You can duplicate the same and rename it to .env. It'll connect to your local MongoDB

**Comment by web-dev-kushal** [comment]
Can you please describe the issue

**Comment by manaschubby** [comment]
Yeah so basically after the patient has been added, we need to reload the patients table.

Right now the new patient doesn't show up on the table unless we reload the page. 

So that has to be fixed

**Comment by manaschubby** [comment]
Hi @web-dev-kushal, were you able to make progress with the issue?
